rpm/python3-sphinx.spec

```diff
@@ -63,7 +63,7 @@ sed '1d' -i sphinx/pycode/pgen2/token.py
 # patch to support this incorporated in 0.6.6
 pushd %{buildroot}%{python3_sitelib}
 
-for lang in `find sphinx/locale -maxdepth 1 -mindepth 1 -type d -not -path '*/\.*' -printf "%f "`;
+for lang in `find sphinx/locale -maxdepth 1 -mindepth 1 -type d -not -path '*/\.*' -exec basename "{}" ";" | tr "\n" " "`;
```

Comment:
Replace the backticks here with $().

Reply:
Why? Please note that I didn't introduce the backticks, nor are they related to the task at hand.

Reply:
I know that you didn't introduced them.
Why backticks should be replaced with $( ): 
- https://www.shellcheck.net/wiki/SC2006
- http://mywiki.wooledge.org/BashFAQ/082

In this case it doesn't matter as much but $() ist just easier to read.

Reply:
I'm well aware of the generic reasons why you shouldn't use backticks. The reasons just don't really apply here - There is no nesting, no escaping, no undefined behaviour. I don't really agree even with "easier to read" in this case, as the backticks are just as easy. That's just my opinion though. Nevertheless, I'm not eager to change code which was probably written by someone smarter than me, unless there is a reason that really applies to this case.

Reply:
agreed it's not needed as such but I too think it makes sense to clean up spec files when we notice issues and $() is much more obvious given the nested `''""` quotes :)

Reply:
> agreed it's not needed as such but I too think it makes sense to clean up spec files when we notice issues and $() is much more obvious given the nested `''""` quotes :)

I think it's originally your code (or you borrowed it from Fedora), so if you say it should be changed I'll change it :)